packages/symantec_endpoint_security/data_stream/incident/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,470 @@
+---
+description: Pipeline for processing Incident logs.
+processors:
+  - set:
+      field: ecs.version
+      tag: set_ecs_version
+      value: 8.11.0
+  - fail:
+      if: ctx.error?.message != null && ctx.message == null && ctx.event?.original == null
+      message: error message set and no data to process
+  - rename:
+      field: message
+      target_field: event.original
+      tag: rename_message
+      ignore_missing: true
+      if: ctx.event?.original == null
+  - json:
+      field: event.original
+      tag: json_decoding
+      target_field: ses.incident
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - fingerprint:
+      fields:
+        - ses.incident.incident_uid
+        - ses.incident.time
+      tag: fingerprint_processor
+      target_field: _id
+      ignore_missing: true
+  - set:
+      field: event.kind
+      tag: set_event_kind
+      value: alert
+  - append:
+      field: event.category
+      tag: append_malware_into_event_category
+      value: malware
+      if: >-
+        ctx.ses?.incident?.conclusion != null && [
+          'targeted malware',
+          'suspicious activity',
+          'malicious activity',
+          'malicious file',
+          'dual use tool',
+          'malware',
+          'hacktool'
+        ].contains(ctx.ses.incident.conclusion.toLowerCase())
+  - append:
+      field: event.category
+      tag: append_threat_into_event_category
+      value: threat
+      if: >-
+        ctx.ses?.incident?.conclusion != null && [
+          'network threat',
+          'potential threat',
+          'targeted attack'
+        ].contains(ctx.ses.incident.conclusion.toLowerCase())
+  - append:
+      field: event.category
+      tag: append_intrusion_detection_into_event_category
+      value: intrusion_detection
+      if: >-
+        ctx.ses?.incident?.conclusion != null && [
+          'breach'
+        ].contains(ctx.ses.incident.conclusion.toLowerCase())
```

If we're not expecting more categories here...
```suggestion
      if: ctx.ses?.incident?.conclusion?.equalsIgnoreCase('breach') == true
```